app/vmalert/remotewrite/remotewrite.go

```diff
@@ -223,11 +227,11 @@ func (c *Client) flush(ctx context.Context, wr *prompbmarshal.WriteRequest) {
 		attempts, len(wr.Timeseries))
 }
 
-func (c *Client) send(ctx context.Context, data []byte) error {
+func (c *Client) send(ctx context.Context, data []byte) (bool, error) {
```

Comment:
wdyt about introducing retriable type of error? This could simplify the code and bring less changes into it

Reply:
Yeah we could do that, ill prepare a suggestion tomorrow!

Reply:
added a retriableError type